src/providers/export/azureCustomVision.test.ts

```diff
@@ -68,7 +72,10 @@ describe("Azure Custom Vision Export Provider", () => {
         };
 
         testProject.exportFormat.providerOptions = customVisionOptions;
-        const provider = new AzureCustomVisionProvider(testProject, testProject.exportFormat.providerOptions);
+        const provider = new AzureCustomVisionProvider(
+            testProject,
+            testProject.exportFormat.providerOptions as IAzureCustomVisionExportOptions,
+        );
```

Comment:
Nit: should extract this to a helper function, seeing repeating code in other tests.

Reply:
Will attempt to move this code up